source/ddbus/conv.d

```diff
@@ -7,7 +7,7 @@ import ddbus.util;
 import ddbus.thin;
 
 import std.exception : enforce;
-import std.meta: allSatisfy;
+import std.meta: allSatisfy, Filter;
```

Comment:
any reason for this import addition?

Reply:
Nope, looks like I forgot to revert that... I used it in my first attempt to fix this...